### ISSUE TITLE
fatal error: 'jpeglib.h' file not found and opencv/cv.h file not found in windows 8.1 then gcc not a valid win32 application

### ISSUE THREAD

**Issue by Madhu172006** [gen2brain/cam2ip#9]
![mingw64](https://user-images.githubusercontent.com/39514026/44853204-ee632100-ac82-11e8-85ca-bbd9eb4a6ed8.jpg)
![image](https://user-images.githubusercontent.com/39514026/44853212-f28f3e80-ac82-11e8-8cdd-2e7aa2bcc73d.png)

**Comment by gen2brain** [comment]
Yes, libjpeg-turbo (optional) and opencv are requirements, as noted in README,

**Comment by Madhu172006** [comment]
actually i had this installed and set the paths to their bin folders but still i get that error

**Comment by gen2brain** [comment]
You then need to set CGO_CFLAGS/LDFLAGS to your includes/libraries directories so they can be found, use go build in verbose mode to see what is happening.
